Gemfile

```diff
@@ -5,6 +5,9 @@ ruby '2.5.1'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.1'
+gem 'figaro'
```

Comment:
I remember I heard you talking about this gem earlier today with Trino. Is necessary to add a gem to manage environment variables or does rails have a way to manage them internally?

Reply:
Archss use an environment variable with ruby but when he tried to do the send email he can't. This is the reason why i used the Figaro and the deadline too.